MealFridge/Models/Partials/RecipePartial.cs

```diff
@@ -1,12 +1,19 @@
 ﻿using MealFridge.Models.Interfaces;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace MealFridge.Models
 {
     public partial class Recipe : IFoodItem
     {
+       
```

Comment:
Why all of this?

Reply:
Added partial so that I could flag a bool which helped me complete my third story with restrictions being accounted for in the search recipes function.